TestCases/coupled_cht/disc_adj_unsteadyCHT_cylinder/postprocess.py

```diff
@@ -0,0 +1,28 @@
+# Compute and print absolute difference between Discrete Adjoint
+# and Finite Difference gradient. Prints also percentage difference.
+
+import pandas as pd
+
+# load files
+DAgrad = pd.read_csv("of_grad.csv")
```

Comment:
you could setup the fin diff script to compute the gradient and do everything without reading files
that would make the script almost ready to run an optimization.

Reply:
Good point ... didn't think of that. Make it `gradientValidation.py` and ditch the `postprocess.py`

Reply:
I added the adjoint run such that the FADO script can run the gradient validation without any by-hand modifications to the stock git files. I kept the `postprocess.py` file though as I think having that completely separate is nice as one can compute once and then play around with the output a bit.